server/graphql/v1/mutations/collectives.js

```diff
@@ -462,13 +467,7 @@ export async function approveCollective(remoteUser, CollectiveId) {
   });
 
   // Approve all events created by this collective under this host
-  models.Collective.findAll({
-    where: {
-      type: types.EVENT,
-      HostCollectiveId: host.id,
-      ParentCollectiveId: collective.id,
-    },
-  }).then(events => {
+  collective.getEvents({ where: { HostCollectiveId: host.id } }).then(events => {
```

Comment:
I know it was there before but I don't like the idea of this condition ```{ where: { HostCollectiveId: host.id } }```. Should we remove it?

Reply:
Also, should we `await`? And refactor using `await` instead of `then`?

Reply:
> I know it was there before but I don't like the idea of this condition `{ where: { HostCollectiveId: host.id } }`. Should we remove it?

Yes, I just verified in the DB and it is safe to assume that, for the current data set, the event host is always equal to the collective host.

> Also, should we `await`? And refactor using `await` instead of `then`?

I didn't do it for performance issues, I assumed that was the reason why this promised is being called in parallel.